File: tests/performance/c/fib.c
#include <stdlib.h>
#include <stdio.h>

static int fib(int i) {
	if(i == 1) {
		return 1;
	}
	if(i == 2) {
		return 1;
	}
	return fib(i-1) + fib(i-2);
}

int main(int argc, char const *argv[]) {
	int n = atoi(argv[1]);
	printf("%d\n", fib(n));
	return 0;
}

File: tests/performance/java/Fib.java
public class Fib {
	private static int fib(int i) {
		if(i == 1) {
			return 1;
		}
		if(i == 2) {
			return 1;
		}
		return fib(i-1) + fib(i-2);
	}	

	public static void main(String[] args) {
		int n = Integer.parseInt(args[0]);
		System.out.println(fib(n));
	}
}